packages/maskbook/src/plugins/Collectible/hooks/useAsset.ts

```diff
@@ -90,7 +90,7 @@ export function useAsset(provider: CollectibleProvider, token?: CollectibleToken
                     is_verified: false,
                     is_owner: false,
                     is_auction: false,
-                    image_url: raribleResponse.properties.imagePreview,
+                    image_url: toRaribleImage(raribleResponse.properties.image),
```

Comment:
Please test [the issue](https://github.com/DimensionDev/Maskbook/issues/3122). 
Sometimes rarible doesn't return the `image`

Reply:
Sometimes no imagePreview either.

Reply:
I changed to use `imagePreview`, if it's not provided, then use `image`